region/brinstar/green/Green Brinstar Main Shaft.json

```diff
@@ -799,6 +802,66 @@
       "notable": false,
       "requires": []
     },
+    {
+      "link": [9, 9],
+      "name": "Leave With Runway",
+      "notable": false,
+      "requires": [],
+      "exitCondition": {
+        "leaveWithRunway": {
+          "length": 4,
+          "openEnd": 1
+        }
+      }
+    },
+    {
+      "link": [9, 9],
+      "name": "Leave Shinecharged",
+      "notable": false,
+      "requires": [
+        {"canShineCharge": {
+          "usedTiles": 17,
+          "openEnd": 0
```

Comment:
```suggestion
          "openEnd": 1
```

Reply:
There is a difference between running into the wall and stopping short.  It'd be nice to represent it somehow,
~135 frames stopping early
~120 running into wall
my best times.

Reply:
He was asking me about this one, and because of the need for a moonfall, you can't fall off. Were you doing it by running to the left?

Reply:
Done. I just created an alternate strat with a one-tile shorter runway, which I think can represent this fairly well.